backend/src/packages/chaiNNer_onnx/__init__.py

```diff
@@ -2,6 +2,23 @@
 
 from api import KB, MB, Dependency, add_package
 from gpu import nvidia_is_available
+from system import is_arm_mac
+
+if is_arm_mac:
+    package_description = (
+        "ONNX uses .onnx models to upscale images. It also helps to convert between"
+        " PyTorch and NCNN. However, it does not support CoreML."
+    )
+    inst_hint = "ONNX uses .onnx models to upscale images."
+else:
+    package_description = (
+        "ONNX uses .onnx models to upscale images. It also helps to convert between"
+        " PyTorch and NCNN. It is fastest when CUDA is supported. If TensorRT is"
+        " installed on the system, it can also be configured to use that."
+    )
+    inst_hint = (
+        "ONNX uses .onnx models to upscale images. It does not support AMD GPUs."
+    )
```

Comment:
There is a lot of redundancy here. Could we maybe extract the bits that are same out into variables? Same for PyTorch and NCNN.

Reply:
I will remove the redundancy.